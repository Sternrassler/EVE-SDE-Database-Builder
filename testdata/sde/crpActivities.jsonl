{"activityID":1,"activityName":"Manufacturing","iconNo":"07_15","description":"Manufacturing items from blueprints"}
{"activityID":3,"activityName":"Research Time Efficiency","iconNo":"07_03","description":"Researching time efficiency"}
{"activityID":4,"activityName":"Research Material Efficiency","iconNo":"07_03","description":"Researching material efficiency"}
