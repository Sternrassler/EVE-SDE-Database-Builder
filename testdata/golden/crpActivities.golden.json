[
  {
    "activityID": 1,
    "activityName": "Manufacturing",
    "description": "Manufacturing items from blueprints"
  },
  {
    "activityID": 3,
    "activityName": "Research Time Efficiency",
    "description": "Researching time efficiency"
  },
  {
    "activityID": 4,
    "activityName": "Research Material Efficiency",
    "description": "Researching material efficiency"
  }
]